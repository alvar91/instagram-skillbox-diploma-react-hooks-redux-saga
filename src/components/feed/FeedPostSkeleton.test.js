// Core
import React from "react";
import renderer from "react-test-renderer";
import FeedPostSkeleton from "./FeedPostSkeleton";

describe("FeedPostSkeleton", () => {
  it("FeedPostSkeleton correctly renders markup", () => {
    const tree = renderer
      .create(
        <FeedPostSkeleton />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
