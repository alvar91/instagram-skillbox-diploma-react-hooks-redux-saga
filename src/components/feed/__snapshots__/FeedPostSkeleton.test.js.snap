// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeedPostSkeleton FeedPostSkeleton correctly renders markup 1`] = `
<div
  className="makeStyles-container-1"
>
  <div
    className="makeStyles-headerSkeleton-2"
  >
    <div
      className="makeStyles-avatarSkeleton-3"
    />
    <div
      className="makeStyles-headerTextSkeleton-4"
    >
      <div
        className="makeStyles-primaryTextSkeleton-5"
      />
      <div
        className="makeStyles-secondaryTextSkeleton-6"
      />
    </div>
  </div>
  <div
    className="makeStyles-mediaSkeleton-7"
  />
</div>
`;
